CHANGELOG.md

```diff
@@ -1,5 +1,75 @@
 # [Versions](https://mui.com/versions/)
 
+## 5.10.17
+
+<!-- generated comparing v5.10.16..master -->
+
+_Dec 5, 2022_
+
+A big thanks to the 15 contributors who made this release possible. Here are some highlights ✨:
+
+- @mnajdova added a [Material You Button playground](https://mui.com/material-ui/react-button/#material-you-version) (#35222)
+- 🔧 @hbjORbj renamed `components` / `componentProps` to `slots` / `slotProps` prop in Joy UI to create consistency across products (#34997)
+- Many other 🐛 bug fixes, 📚 documentation, and ⚙️ infrastructure improvements
+
+### `@mui/material@5.10.17`
+
+- &#8203;<!-- 28 -->[material-ui] Add channel colors if possible (#35178) @siriwatknp
+- &#8203;<!-- 10 -->[Fab] Increase disabled styles precedence (#35304) @Uzwername
+- &#8203;<!-- 05 -->[Rating] Apply `labelEmptyValueActive` style overrides from theme (#35315) @sai6855
+
+### `@mui/system@5.10.17`
+
+- &#8203;<!-- 04 -->[system] Add support for nested CssVarsProvider (#35277) @siriwatknp
+
+### `@mui/joy@5.0.0-alpha.57`
+
+#### BREAKING CHANGE
+
+- &#8203;<!-- 08 -->[Joy] Add `slots`/`slotProps` props to the typing of all components and apply `useSlot` to all components (#34997) @hbjORbj
+
+  - Change all occurrences of `components` and `componentsProps` props in Joy UI components to `slots` and `slotProps`, respectively.
+  - Change casing of slots' fields to camelCase
+
+  ```diff
+  -<Autocomplete components={{Root: CustomRoot}} componentsProps={{input: { className: 'custom-input' }}} />
+  +<Autocomplete slots={{root: CustomRoot}} slotProps={{input: { className: 'custom-input' }}} />
+  ```
```

Comment:
Don’t we have a codemod? If yes we should mention it here

Reply:
- `@mui-codemod` [README](https://github.com/mui/material-ui/blob/master/packages/mui-codemod/README.md#joy-rename-components-to-slots) explains how to run the appropriate codemod for this breaking change.
  
  Does this seem enough?

Reply:
Let’s link the codemod in the breaking changes section, this is what I meant :)

Reply:
I made this [commit](https://github.com/mui/material-ui/pull/35354/commits/e157aeba5cd8d006a3d71add10b80f5043d0c47e). Let me know if that's what you meant.